tests/test_contains.py

```diff
@@ -43,10 +30,56 @@ def test_includes():
   ip2 = IPRange(group=group2, value="10.204.145.0")
   ip2.save()
   ip = "10.204.145.0"
-  relation1 = Relation(subject=group1, object=group2, relation=Relation.RelationType.INCLUSION)
+  relation1 = Relation(subject=group1, object=group2, relation=Relation.RelationType.EXCLUSION)
   relation1.save()
   
+  assert not ip in group1
```

Comment:
This should be rewritten as `assert ip not in group1`. (See also https://docs.python.org/3/reference/expressions.html#not-in)